VimeoUpload.podspec

```diff
@@ -21,7 +21,9 @@ Pod::Spec.new do |s|
   s.osx.deployment_target = '10.9'
 
   s.source       = { :git => "https://github.com/vimeo/VimeoUpload.git", :tag => s.version.to_s }
-  s.source_files  = "VimeoUpload", "VimeoUpload/**/*.{swift}"
+  s.source_files  = "*.{h}", "VimeoUpload", "VimeoUpload/**/*.{swift}", "VimeoUpload/**/*.{h}"
```

Comment:
I don't think the changes to the podspec are necessary, we're actually not going to be able to make the library available as a pod until the Vimeo iOS app drops iOS7 support (sometime this summer probably).